### DISCUSSION TITLE
Add artisan command to generate a new table

### DISCUSSION THREAD

**Discussion by iAmKevinMcKee** [rappasoft/laravel-livewire-tables#399] [Feature Requests]
Would you be interested in a `php artisan make:livewire-datatable UsersTable` command that would scaffold out the livewire component including the extension of the datatable class?

**Comment by rappasoft** [comment]
I would. It's on my list but I haven't decided what options it's going to have yet.
 Probably `php artisan make:table` unless it conflicts with something else.

**Reply by iAmKevinMcKee** [reply]
Looks good to me. I would think to start just add the most basic class and view, which is tremendously valuable in my opinion. You could add a bunch more configurable stuff, but I think just the basic would be really helpful. I went digging through the docs earlier because I thought for sure I had used a command to generate a previous table, but then realized it isn't there.

I'm happy to create a PR with the basic functionality if you're interested.

**Reply by rappasoft** [reply]
Yeah definitely start with whatever you think is right and I can build on it if needed. Thanks!

**Reply by iAmKevinMcKee** [reply]
Submitted #408

Let me know what you think.
